langserver/langserver.py

```diff
@@ -35,6 +36,9 @@ def run(self):
             self.handle(request)
 
     def handle(self, request):
+
+        handler_span = Tracer.start_span(request["method"])
```

Comment:
if this throws an exception or returns early that isn't recorded. spans are context managers, so you should be able to use a `with` statement. We don't really want to indent this whole block of code, so maybe some refactoring of the actual logic into its own function vs the instrumentation.

Also if `method` is not in the request, this will throw an exception. Use `request.get('method', 'UNKNOWN')`